docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -1,5 +1,6 @@
 ---
-description: Managing credentials
+description: Find out how to manage credentials with Kargo
```

Comment:
This description is a little bit misleading. Kargo does not (yet) have any capability for managing credentials.

The content of this document is really about managing credentials (using kubectl) for use _by_ Kargo.

My suggestion:

```suggestion
description: Find out how to manage repository credentials for use by Kargo
```